app/assets/stylesheets/tables-report.css.scss

```diff
@@ -15,3 +25,20 @@
     }
   }
 }
+
+.usa-table {
+  tr th,
+  tr td {
+    border: 2px solid $secondary-color;
+  }
+
+  thead th[scope='col'] {
+    background-color: $secondary-color;
```

Comment:
To limit the number of nonstandard styles applied to these tables, would consider using as many themeable design system variables as possible for these customizations, e.g. `$theme-table-header-background-color`, `$theme-table-border-color`, `$theme-table-background-color`, etc.

https://designsystem.digital.gov/components/table/#component-settings-table